pkg/apis/core/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -269,7 +268,7 @@ func (in *FederatedClusterSpec) DeepCopyInto(out *FederatedClusterSpec) {
 	out.ClusterRef = in.ClusterRef
 	if in.SecretRef != nil {
 		in, out := &in.SecretRef, &out.SecretRef
-		*out = new(v1.LocalObjectReference)
+		*out = new(LocalSecretReference)
 		**out = **in
```

Comment:
I assume this is generated code.

Reply:
@xunpan it's not generated code. I'd manually changed this to make it work.

Reply:
@sohankunkerkar This should be generated code. Let's not manually change it.

Reply:
ack